server/src/main/java/org/elasticsearch/rest/action/cat/AbstractCatAction.java

```diff
@@ -42,6 +42,11 @@
 
     protected abstract Table getTableWithHeader(RestRequest request);
 
+    @Override
+    public boolean allowSystemIndexAccessByDefault() {
+        return true;
```

Comment:
why do we allow system by default for cat apis?

Reply:
This ties into the "Resolving `_all` causes a deprecation warning" issue, and we can't omit them (like we do with hidden indices) in 7.x, as that would break BWC.